### ISSUE TITLE
status: "Data and pipelines are up to date." when using --cloud

### ISSUE THREAD

**Issue by ghost** [iterative/dvc#1618]
I'm not sure if `Pipeline is up to date. Nothing to reproduce.` is the best message to describe the `status` of a remote storage.

When using `fetch/pull/push` you are not **reproducing** anything, you are **syncing** the cache.

I would suggest something more _ad hoc_ like the following: `Cache is up to date. Nothing to sync with '{}' remote`.

Also, it can be confusing if `dvc status` shows you some "changes" and `dvc status -c` outputs "Nothing to reproduce"

**Comment by efiop** [comment]
Related https://github.com/iterative/dvc/issues/1202

**Comment by jorgeorpinel** [comment]
@mroutis I believe the message has changed at this point, does this issue desc. need an update? Thanks

**Comment by efiop** [comment]
@jorgeorpinel indeed current message is
```
➜  dvc git:(master) ✗ dvc status -c   
Data and pipelines are up to date.
```
But the gist of the issue stays the same. Updated the desc though. Thank you for the heads up! 🙂

**Comment by dashohoxha** [comment]
Maybe the best solution is to rename `dvc status --cloud` to `dvc remote status` or `dvc remote diff`. Then it is going to have a completely different set of messages from `dvc status`.

I think this idea (splitting `dvc status` and `dvc status -c` into two different commands) has been proposed previously, but I couldn't find the relevant issues.

**Comment by kurianbenoy** [comment]
Can I reserve this issue to be used for today's Devsprints?

**Comment by efiop** [comment]
@kurianbenoy Sure!

**Comment by tenitski** [comment]
Looking at this issue 👀

**Comment by efiop** [comment]
Fixed by #4694
